### PR TITLE
Job request updated

### PR DESCRIPTION
**Pull request by Fewwy**

### CODE REVIEW

File: app/routes/async/job.test.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        "urls": [
+          {"url": "http://example.org/items", "params": { "testParam": "testParamValue", "page": 1 }},
+          {"url": "http://example.org/items", "params": { "testParam": "testParamValue", "page": 2 }}
+        ]
+      }
+    test('compare the job requests length', async () => {
+        const jobsAmount = await prisma.job.findMany()
+        loader({ data: { request :
+            body
+        }});
```
[the remark this answers is not in the document]
Comment (Fewwy) [reply app/routes/async/job.test.js:15]: This results in an error 
image

File: app/routes/async/job.jsx [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import { json } from "@remix-run/node";
 import { create } from '../../models/job.server'
 
-export const loader = async () => {
-  // handle "GET" request
-  const response = await create()
+export const loader = async ({request}) => {
+  const response = await create(request)
```
Comment (bastilian) [inline app/routes/async/job.jsx:5]: This is only passing the `request` property of the request object along, which will likely not work.

line 4 should have `({request: {body}}`

With that you can access body.urls.
Create a normal job without passing anything along, use the id of that created job and create the job requests by iterating over the `body.urls`, like here: https://github.com/bastilian/AAAPI/blob/main/app/models/job_request.server.test.js#L8

File: app/models/job_request.server.js [hunk 3]
```diff
@@ -9,10 +9,10 @@ export const getById = async (id) =>
   })
 );
 
-export async function create(data) {
-  return prisma.jobRequest.create({
+export async function create(data = {}) {
+  return prisma.jobRequest.create(
```
Comment (bastilian) [inline app/models/job_request.server.js:13]: This is not right. The `create` function expects a `data` property.

Comment (bastilian) [reply app/models/job_request.server.js:13]: It was right before. :)

**Review by adonispuente — DISMISSED** [review]
LGTM!

File: app/models/job_request.server.js (hunk 3 above)
Comment (bastilian) [reply app/models/job_request.server.js:13]: Please keep the format as before.

**Pull request merged**